modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/BuildOutlet.tsx

```diff
@@ -126,13 +125,13 @@ const BuildOutlet: React.FC<BuildOutletProps> = ({ignorePaths}) => {
 		}
 	}, [basePath, isCurrentPathIgnored, pathname, setTabs]);
 
-	if (testrayBuild) {
-		return (
-			<>
-				{!isCurrentPathIgnored && (
-					<BuildOverview testrayBuild={testrayBuild} />
-				)}
+	return (
+		<>
+			{!isCurrentPathIgnored && testrayBuild && (
+				<BuildOverview testrayBuild={testrayBuild} />
+			)}
```

Comment:
Move this inside PageRenderer, before the Outlet
```suggestion
	
```

Reply:
Done